lib/api.js

```diff
@@ -1476,6 +1476,26 @@ module.exports = function(self, options) {
     }
   };
 
+  self.revert = function(req, commitId, callback) {
+     var id = self.apos.launder.id(commitId);
+
+     return async.waterfall([getDocAndCommit, copyIncludedProperties, update], callback);
+    
+     function getDocAndCommit(cb) {
+       self.findDocAndCommit(req, id, cb);
```

Comment:
Every invocation of an async function should always use the `return` keyword (unless you are intentionally falling through and doing two things at once, which is rare and should be labeled as such with a comment).

Going to be a stickler for this from here on out.

Reply:
Sounds a lot like intentionally falling through a roof - I'm too old for that ;)